spmd/tensor/ops/pointwise_ops.py

```diff
@@ -0,0 +1,116 @@
+from typing import List, Optional
+from spmd.tensor.placement_types import Shard, Replicate, _Partial, PlacementSpec
+
+[
```

Comment:
QQ, how do we come up with this list and ensure its coverage?

Reply:
It's coming from https://github.com/pytorch/pytorch/blob/fd84c458f49df2ec2801a230f332ea3730d352a9/docs/source/torch.rst#L275

It seems that it's still missing other pointwise ops as it seems not a complete list. To ensure test coverage, I guess we need to loop through the list and compare them with local tensor operations